app/bot.go

```diff
@@ -193,6 +193,93 @@ func (a *App) disableUserBots(userId string) *model.AppError {
 	return nil
 }
 
+func (a *App) notifySysadminsBotDisabled(userId string) *model.AppError {
+
+	disableBotsSetting := *a.Config().ServiceSettings.DisableBotsWhenOwnerIsDeactivated
+
+	perPage := 20
+	options := &model.BotGetOptions{
```

Comment:
We don't actually retrieve a full list here only the first 20. Should we get all the user's bots for the message? If not we should let the admin know there may be more.

Reply:
@jasonblais, what do you think about limiting the number of admins notified. I would lean toward notifying all admins.

@crspeller, the PerPage field in the options struct maps to the LIMIT value in the SQL statement. What do you think about setting this to something like 1000?

Reply:
Is this about limiting # of admins being notified, or the bots being shared with the admin?

I'd prefer not limiting admins (given we might exclude the one who is the acting admin at that time), but good to limit bots to 10 and modify message accordingly to make it clear there are more.

Reply:
@jasonblais The `PerPage` limit is actually set to 20 for both # of user bots and # of admins notified.  

For this particular `PerPage` setting, we are looking at the number of bots that will be listed.  If we set an upper limit to the number of bots shown in the bulleted list, we will need to modify the message to let the admin know that there are more bots owned by the user, but not shown in the list.

Reply:
@jfrerich We can just modify `The user managed the following bot accounts which have now been disabled` to `The user managed XX bot accounts including the following, which have now been disabled`

And if we can't easily retrieve the number of bot accounts, `The user managed several bot accounts bot accounts including the following, which have now been disabled` is also fine

Reply:
@jfrerich I would rather set perPage to a reasonable number and have a separate count query to get the number of bots the admin has. Better than setting a random large value.

Reply:
@crspeller, that makes sense.  I'll loop through the reduce the `perPage` value and loop through the pages, while keep count.